src/ngx_http_brotli_static_module.c

```diff
@@ -322,16 +322,18 @@ static ngx_int_t
 ngx_http_brotli_accept_encoding(ngx_str_t *ae)
 {
     u_char  *p;
+    char* encoding = "br";
+    size_t encoding_length = ngx_strlen("br");
```

Comment:
Maybe we can switch back to `sizeof` instead of recomputing it every time?

Reply:
Let me check the compiled code. I believe, compiler should be smart enough to avoid the computation.

Reply:
```
_ngx_http_brotli_accept_encoding:
     560:       55      pushq   %rbp
     561:       48 89 e5        movq    %rsp, %rbp
     564:       53      pushq   %rbx
     565:       50      pushq   %rax
     566:       48 89 fb        movq    %rdi, %rbx
     569:       48 8b 7f 08     movq    8(%rdi), %rdi
     56d:       48 8d 35 f7 02 00 00    leaq    759(%rip), %rsi
     574:       ba 02 00 00 00  movl    $2, %edx  # <<== here
     579:       e8 00 00 00 00  callq   0 <_ngx_http_brotli_accept_encoding+0x1e>
     57e:       48 89 c1        movq    %rax, %rcx
     581:       48 c7 c0 fb ff ff ff    movq    $-5, %rax
     588:       48 85 c9        testq   %rcx, %rcx
```

Reply:
Nice, thanks for checking.

Reply:
https://godbolt.org/z/D_8DmA